.github/workflows/main.yml

```diff
@@ -106,7 +106,7 @@ jobs:
       - name: create target branch
         id: target_create
         if: ${{ steps.target_get.outcome == 'failure' }}
-        uses: peterjgrainger/action-create-branch@v2.0.1
+        uses: peterjgrainger/action-create-branch@v3.0.0
```

Comment:
This should fix our issues creating the bundled branches. v2.0.1 is 4 years old and the GitHub API has slightly tweaked how it handles non-existent branches since then. v3 appears to fix it.